pkg/kubelet/server/server.go

```diff
@@ -941,14 +939,21 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 	longRunning := strconv.FormatBool(isLongRunningRequest(path))
 
-	servermetrics.HTTPRequests.WithLabelValues(method, path, serverType, longRunning).Inc()
-
 	servermetrics.HTTPInflightRequests.WithLabelValues(method, path, serverType, longRunning).Inc()
 	defer servermetrics.HTTPInflightRequests.WithLabelValues(method, path, serverType, longRunning).Dec()
 
 	startTime := time.Now()
 	defer servermetrics.HTTPRequestsDuration.WithLabelValues(method, path, serverType, longRunning).Observe(servermetrics.SinceInSeconds(startTime))
 
+	handler := httplog.WithLogging(s.restfulCont, statusesNoTracePred, func(req *http.Request, status int, hijacked bool) {
+		code := strconv.Itoa(status)
+		if hijacked {
+			// the caller take over the connection, we don't know the status.
+			code = "unknown"
```

Comment:
some of the backends use typed values for metric labels, so we should use something like -1.

Reply:
All right, thanks for pointing out. :)